src/oop/user_defined_classes/medium/q2/QuestionText.md

```diff
@@ -0,0 +1,29 @@
+# oop-user_defined_classes-medium-q2
+
+- Java
+
+Little boys love to play with cars and trucks and watch things move. 
+Make a Java class called Car that has attributes called colour, 
+horsePower, and yearMade. This class also has a constructor for 
+initializing the attributes and another method called printSpecs() 
+that displays the colour and horse power of the vehicle.
+
+
+### Car Class
+
+Your `Car` class should have the following items:
+
+- Attributes:
```

Comment:
Update this so that they know better what they need to code

Reply:
For example, you could have something more along the lines of:

The `Car` class you are given has the following items: 

whatever you give them

You must add the following items to the class:

x